'Solution'

Irina17191 opened the pull request.

Irina17191 committed:
'Solution'

Arsen-hrynevych commented on cinema/serializers.py:
```diff
@@ -0,0 +1,18 @@
+from rest_framework import serializers
+from cinema.models import Movie
```
(

Arsen-hrynevych commented on cinema/urls.py:
```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+from cinema.views import movie_list, movie_detail
```
(

Arsen-hrynevych reviewed (changes requested):
А де весь проект? Просто я бачу в тебе в репозиторії лише 2 файли

Irina17191 committed:
'Solution'

Irina17191 commented:
> А де весь проект? Просто я бачу в тебе в репозиторії лише 2 файли

I'm sorry, my bad( I don't know how I didn't notice that.

Also now I can see that here __pycache__ files have been added to git, despite I have added these files to .gitignore, I will  figure out with it

update: I deleted __pycache__ files from git

Irina17191 committed:
'Solution'

maxgerman commented on cinema/views.py:
```diff
@@ -0,0 +1,39 @@
+        return Response(serializer.data, status=status.HTTP_200_OK)
+    if request.method == "PUT":
+        serializer = MovieSerializer(movie, data=request.data)
+        if serializer.is_valid():
```
how can we avoid this `if` (and similar) and keep the same functionality?

Irina17191 replied:
by using raise_exception=True, thank you, fixed it)

maxgerman commented on cinema/urls.py:
```diff
@@ -0,0 +1,10 @@
+
+
+urlpatterns = [
+    path("movies", movie_list, name="movie_list"),
```
see the checklist:
- endpoints should end with the slash
- hyphen should be the delimiter in the url names

Irina17191 replied:
Thank you)

maxgerman reviewed (changes requested):
Good job!
See some comments

Irina17191 committed:
'Solution'

Arsen-hrynevych commented on app/settings.py:
```diff
@@ -0,0 +1,125 @@
+    "django.contrib.sessions",
+    "django.contrib.messages",
+    "django.contrib.staticfiles",
+    "cinema",
+    "rest_framework",
```
Are you sure about ordering

Arsen-hrynevych reviewed (approved):
Thanks
